openslides/assignments/static/templates/assignments/assignment-detail.html

```diff
@@ -24,18 +24,98 @@
+  <alert ng-show="alert.show" type="{{ alert.type }}" ng-click="alert={}" close="alert={}">
+    {{alert.msg}}
+  </alert>
+  <div os-perms="assignments.can_nominate_other" class="input-group">
+    <ui-select ng-model="candidate.selected" ng-change="addCandidate(candidate.selected.id)">
+      <ui-select-match placeholder="{{ 'Select or search a participant...' | translate }}">
+        {{ $select.selected.get_full_name() }}
+      </ui-select-match>
+      <ui-select-choices repeat="user in users | filter: $select.search">
+        <div ng-bind-html="user.get_full_name() | highlight: $select.search"></div>
+      </ui-select-choices>
+    </ui-select>
+    <span class="input-group-btn">
+      <a ng-click="candidate={}" class="btn btn-default">
+        <i class="fa fa-times-circle"></i>
+      </a>
+    </span>
+  </div>
+  <p os-perm="assignments.can_nominate_self">
+    <button ng-if="!isCandidate()" ng-click="addMe()" class="btn btn-default">
+      <i class="fa fa-plus"></i>
+      <translate>Add me</translate>
+    </button>
+    <button ng-if="isCandidate()" ng-click="removeMe()" class="btn btn-default">
+      <i class="fa fa-minus"></i>
+      <translate>Remove me</translate>
+    </button>
+</div>
 
 <h3 translate>Election result</h3>
-<!-- TODO -->
+<button os-perms="assignments.can_manage" ng-click="createBallot()" class="btn btn-default btn-sm">
+  <i class="fa fa-bar-chart fa-lg"></i>
+  <translate>New ballot</translate>
+</button>
+
+<uib-tabset class="spacer">
+  <uib-tab ng-repeat="poll in assignment.polls" heading="Ballot {{$index+1}}">
+    <div os-perms="assignments.can_manage" class="spacer">
+      <button ng-click="editPollDialog(poll)"
+          class="btn btn-default btn-sm">
+        <i class="fa fa-pencil"></i>
+        <translate>Edit</translate>
+      </button>
+      <!-- angular requires to open the link in new tab with "target='_blank'".
+      Otherwise the pdf url can't be open in same window; angular redirects to "/". -->
+      <a ui-sref="assignmentpoll_pdf({poll_pk: poll.id})" target="_blank"
+          class="btn btn-default btn-sm">
+        <i class="fa fa-file-pdf-o"></i> Ballot paper
+      </a>
+      <button os-perms-lite="assignments.can_manage" ng-if="!poll.published" ng-click="publishBallot(poll)"
+          class="btn btn-primary btn-sm">
+        <i class="fa fa-globe"></i>
+        <translate>Publish result</translate>
+      </button>
+      <button os-perms-lite="assignments.can_manage" ng-if="poll.published" ng-click="unpublishBallot(poll)"
+          class="btn btn-default btn-sm">
+        <i class="fa fa-globe"></i>
+        <translate>Unpublish result</translate>
+      </button>
+      <a ng-click="deleteBallot(poll)" class="btn btn-default btn-sm">
+        <i class="fa fa-times"></i>
+        <translate>Delete</translate>
+      </a>
+    </div>
+    <div class="results">
+      <div ng-repeat="option in poll.options">
+        <strong>User#{{ option.candidate_id }}</strong>
```

Comment:
I think we have to create DS-Store classes for option objects. See the assignment_related_user as an example.